ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/VulnReports/VulnReportsPage.tsx

```diff
@@ -55,6 +59,11 @@ function VulnReportsPage() {
         hasNotifierIntegrationReadAccess;
 
     const { reports, isLoading, error: fetchError, fetchReports } = useFetchReports();
+    const { isRunning, runError, runReport } = useRunReport({
+        onCompleted: () => {
+            fetchReports();
+        },
+    });
```

Comment:
`useRunReport` has a `useCallback` dependency on the value passed in to `onCompleted` - could you pass a reference directly to `fetchReports` instead of wrapping in an arrow function to take advantage of the memoization in the hook? It looks like `fetchReports` is a stable function so it should prevent the extra calculations.
```suggestion
    const { isRunning, runError, runReport } = useRunReport({
        onCompleted: fetchReports,
    });
```